cmd/dmverity-vhd/main.go

```diff
@@ -22,15 +22,16 @@ import (
 const usage = `dmverity-vhd is a command line tool for creating LCOW layer VHDs with dm-verity hashes.`
 
 const (
-	usernameFlag      = "username"
-	passwordFlag      = "password"
-	imageFlag         = "image"
-	verboseFlag       = "verbose"
-	outputDirFlag     = "out-dir"
-	dockerFlag        = "docker"
-	tarballFlag       = "tarball"
-	hashDeviceVhdFlag = "hash-dev-vhd"
-	maxVHDSize        = dmverity.RecommendedVHDSizeGB
+	usernameFlag       = "username"
+	passwordFlag       = "password"
+	imageFlag          = "image"
+	verboseFlag        = "verbose"
+	outputDirFlag      = "out-dir"
+	dockerFlag         = "docker"
+	bufferedOpenerFlag = "buffered-opener"
```

Comment:
nit: `buffered-reader`?

Reply:
the sdk we're using calls it `opener`, I think it would be simpler to keep it as that

Reply:
If these are external facing, "reader" makes more sense than "opener".